ag_data/formulas/library/system/mercury_formulas.py

```diff
@@ -0,0 +1,50 @@
+#
+# Formulas for Mercury's built-in and natively supported sensor types
+# Sample processing formulas with primary sensors and fictional Mercury-branded sensors
+#
+
+from ag_data import models
+
+
+def fPass(timestamp, sensor, measurement_payload):
+    result = {}
+
+    return result
+
+
+def fMercurySimpleTemperatureSensor(timestamp, sensor, measurement_payload):
```

Comment:
I think we can change all the `measurement_payload` to `payload` or `params` or `parameters` or `args`. Personally, I feel no information gaining with `measurement_` prefix.

Reply:
Good idea. Will fix.

Reply:
This is now in #376.